internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -462,17 +468,12 @@ public Future<RecordMetadata> delete(
       Callback callback,
       LeaderMetadataWrapper leaderMetadataWrapper,
       DeleteMetadata deleteMetadata) {
-    return delete(key, callback, leaderMetadataWrapper, APP_DEFAULT_LOGICAL_TS, Optional.ofNullable(deleteMetadata));
+    return delete(key, callback, leaderMetadataWrapper, APP_DEFAULT_LOGICAL_TS, deleteMetadata);
   }
 
   @Override
   public Future<RecordMetadata> delete(K key, Callback callback, DeleteMetadata deleteMetadata) {
-    return delete(
-        key,
-        callback,
-        DEFAULT_LEADER_METADATA_WRAPPER,
-        APP_DEFAULT_LOGICAL_TS,
-        Optional.ofNullable(deleteMetadata));
+    return delete(key, callback, DEFAULT_LEADER_METADATA_WRAPPER, APP_DEFAULT_LOGICAL_TS, deleteMetadata);
```

Comment:
This is kind of interesting.  My IDE right now I think would auto format this back to how it was :/

what is our style for this?

Reply:
The new code is shorter, thanks to burning the optional, so it fits on one line now :D